docs/samples/Microsoft.ML.Samples/Dynamic/Calibrator.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Microsoft.ML.Calibrator;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public class CalibratorExample
+    {
+        public static void Calibration()
+        {
+            // Downloading the dataset from github.com/dotnet/machinelearning.
+            // This will create a sentiment.tsv file in the filesystem.
+            // You can open this file, if you want to see the data. 
+            string dataFile = SamplesUtils.DatasetUtils.DownloadSentimentDataset();
+
+            // A preview of the data. 
+            // Sentiment	SentimentText
+            //      0	    " :Erm, thank you. "
+            //      1	    ==You're cool==
+
+            // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
+            // as a catalog of available operations and as the source of randomness.
+            var mlContext = new MLContext();
+
+            // Read the data as an IDataView.
+            var reader = mlContext.Data.CreateTextReader(new TextLoader.Arguments()
+            {
+                Separator = "tab",
+                HasHeader = true,
+                Column = new[]
+                    {
+                        new TextLoader.Column("Sentiment", DataKind.BL, 0),
+                        new TextLoader.Column("SentimentText", DataKind.Text, 1)
+                    }
+            });
+
+            // Read the data
+            var data = reader.Read(dataFile);
+
+            // Split the dataset into two parts: one used for training, the other to train the calibrator
+            var (trainData, calibrateData) = mlContext.BinaryClassification.TrainTestSplit(data, testFraction: 0.1);
```

Comment:
>calibrateData [](start = 28, length = 13)

Do you think "calibratorTrainData" looks better?

Reply:
I think testData as usual looks better, mostly because we don't do any calibration on this Dataview.
Platt calibrator works with scoredData.

---
In reply to: [240316055](https://github.com/dotnet/machinelearning/pull/1816#discussion_r240316055) [](ancestors = 240316055)

Reply:
we score this IDV, than train the calibrator with that scoredData?

---
In reply to: [240343461](https://github.com/dotnet/machinelearning/pull/1816#discussion_r240343461) [](ancestors = 240343461,240316055)